service/src/main/java/bio/terra/workspace/amalgam/tps/TpsApiController.java

```diff
@@ -54,4 +54,10 @@ public ResponseEntity<ApiTpsPaoGetResult> getPao(UUID objectId) {
     ApiTpsPaoGetResult result = tpsApiDispatch.getPao(bearerTokenFactory.from(request), objectId);
     return new ResponseEntity<>(result, HttpStatus.OK);
   }
+
+  @Override
+  public ResponseEntity<Void> clonePao(UUID objectId, UUID cloneObjectId) {
```

Comment:
could you add a test for this method to `TpsBasicPaoTest` where we test other methods in this controller?

Reply:
done